clients/.project

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
looks like this file was added in the wrong place - opened by eclipse or moved from the generated files.

Reply:
Deleted